### PR TITLE
WIP - L1-L2 improvements

### PR DESCRIPTION
**Pull request by uF4No**
WIP

Fixed typos in L1-L2 interoperability page

### CODE REVIEW

**Review by barakshani — APPROVED** [review]

File: docs/dev/developer-guides/bridging/l1-l2-interop.md [hunk 1]
```diff
@@ -17,24 +17,21 @@ This gives the rationale for the new design of the priority queue for zkSync 2.0
 ### How it works in zkSync 1. x
 
 In the previous version of zkSync, we only had two operations that could be sent to zkSync from L1:
-`Deposit` to bridge funds from Ethereum to zkSync and `FullExit`to bridges the funds back from Ethereum (this is essentially the same as `Withdraw` in zkSync 2.0). 
+- `Deposit` to bridge funds from Ethereum to zkSync. 
+- `FullExit` to bridges the funds back from Ethereum (this is essentially the same as `Withdraw` in zkSync 2.0). 
 
-If users wanted to deposit funds to, or withdraw funds from, zkSync, they would have sent a transaction request to the smart contract which then got appended to the deque of 
-priority transactions. The deque has the following rules:
+If users wanted to deposit funds to, or withdraw funds from zkSync, they would have to sent a transaction request to the smart contract which then got appended to the queue of priority transactions. The queue has the following rules:
```
Comment (krofax) [inline docs/dev/developer-guides/bridging/l1-l2-interop.md:23]: Can you rephrase it to:

If users wanted to deposit funds to or withdraw funds from zkSync, they would have to send a transaction request to the smart contract which will then get appended to the queue of priority transactions.

**Review by krofax — APPROVED** [review]
